### PR TITLE
[luci-interpreter] Test quantized ArgMax kernels for ResNet-v1

### PR DESCRIPTION
**Pull request by struss**
Parent Issue: #2730 
Draft PR: #2720

This commit update `ArgMax` operator test case.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
oh, yes. This Case have already uint8 test case, but not using TYPED_TEST. so I refactored previous code.

**Comment by struss** [comment]
I forget to add shape check. I Add it. sorry for late

**Comment by jinevening** [comment]
@binarman @s-barannikov PTAL

File: compiler/luci-interpreter/src/kernels/ArgMax.test.cpp [hunk 1]
```diff
@@ -26,73 +26,71 @@ namespace
 
 using namespace testing;
 
-TEST(ArgMaxTest, Float)
+template <typename T1, typename T2>
+void Check(std::initializer_list<int32_t> input_shape,
+           std::initializer_list<int32_t> dimension_shpae,
```
Comment (binarman) [inline compiler/luci-interpreter/src/kernels/ArgMax.test.cpp:31]: Optional:
typo
```suggestion
           std::initializer_list<int32_t> dimension_shape,
```

File: compiler/luci-interpreter/src/kernels/ArgMax.test.cpp [hunk 2]
```diff
@@ -26,73 +26,71 @@ namespace
...
-  std::vector<int32_t> dimension_data{
-      3,
-  };
-  Tensor input_tensor = makeInputTensor<DataType::FLOAT32>(input_shape, input_data);
-  Tensor dimension_tensor = makeInputTensor<DataType::S32>(dimension_shape, dimension_data);
-  Tensor output_tensor = makeOutputTensor(DataType::S64);
+
+  Tensor input_tensor{getElementType<T1>(), input_shape, {}, ""};
+  input_tensor.writeData(input_data.begin(), input_data.size() * sizeof(T1));
+  Tensor dimension_tensor{DataType::S32, dimension_shpae, {}, ""};
```
Comment (binarman) [inline compiler/luci-interpreter/src/kernels/ArgMax.test.cpp:38]:
```suggestion
  Tensor dimension_tensor{DataType::S32, dimension_shape, {}, ""};
```

File: compiler/luci-interpreter/src/kernels/ArgMax.test.cpp (hunk 1 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/ArgMax.test.cpp:31]: Oh, I will fix it.

File: compiler/luci-interpreter/src/kernels/ArgMax.test.cpp (hunk 2 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/ArgMax.test.cpp:38]: Oh, I will fix it.

**Pull request merged**